OutOfSchool/OutOfSchool.WebApi/Services/GeocodingService.cs

```diff
@@ -140,7 +140,7 @@ public class GeocodingService : CommunicationService, IGeocodingService
                 { "key", config.ApiKey },
                 { "categories", "adr_address" },
                 { "radius", $"{config.Radius}" },
-                { "near", $"{request.Lon},{request.Lat}" }, // it has to be lng first, lat second as per api
+                { "near", $"{request.Lon.ToStringWithDotSeparator()},{request.Lat.ToStringWithDotSeparator()}" }, // it has to be lng first, lat second as per api
```

Comment:
Consider using the CultureInfo or NumberFormatInfo instead of creating an extension method.